src/services/sessions.service.ts

```diff
@@ -23,9 +24,15 @@ export class SessionsService {
 		}
 	}
 
-	static async listByDeploymentId(deploymentId: string): Promise<ServiceResponse<Session[]>> {
+	static async listByDeploymentId(
+		deploymentId: string,
+		sessionState?: ProtoSessionStateType
```

Comment:
We would probably extend this filter soon, wdyt about passing a filter object instead of just the specific filter ?

Reply:
Done